lib/simpleofflinecache.js

```diff
@@ -5,9 +5,35 @@ var cacheHelper = require('sw-cache-helper');
 var debug = 0 ? console.log.bind(console, '[SimpleOfflineCache]') :
  function(){};
 
-function SimpleOfflineCache(cacheName) {
+// Default Match options, not exposed.
+var DEFAULT_MATCH_OPTIONS = {
+  ignoreSearch: false,
+  ignoreMethod: false,
+  ignoreVary: false
+};
+var DEFAULT_MISS_POLICY = 'fetchAndChace';
+// List of different policies
+var MISS_POLICIES = [
+  DEFAULT_MISS_POLICY
+];
+
+
+/**
+ * Constructor for the middleware that serves the content of a
+ * cache specified by it's name.
+ * @param {string} cacheName Name of the cache that will be serving the content
+ * @param {object} [options] Object use to setup the cache matching alternatives
+ * @param {string} [missPolicy] Name of the policy to follow if a request miss
+ *                 when hitting the cache.
+ */
+function SimpleOfflineCache(cacheName, options, missPolicy) {
   this.cacheName = cacheName || cacheHelper.defaultCacheName;
-  this.cache = null;
+  this.options = options || DEFAULT_MATCH_OPTIONS;
+  this.missPolicy = missPolicy || DEFAULT_MISS_POLICY;
```

Comment:
nit: this is totally useless in Chrome (I'm going to report a good reason to get it implemented) so maybe we should warn in case we are in Chrome and we are providing `options`. I'm [working on a polyfill for `sww-raw-cache'](https://github.com/lodr/sww-raw-cache/blob/basic-match-options/lib/sww-raw-cache.js#L69)  but it's not very easy (you need to control how resources are added to the offline cache).